config/config.go

```diff
@@ -56,13 +56,14 @@ type NTPConfig struct {
 
 // EpochStartConfig will hold the configuration of EpochStart settings
 type EpochStartConfig struct {
-	MinRoundsBetweenEpochs int64
-	RoundsPerEpoch         int64
+	MinRoundsBetweenEpochs int
```

Comment:
this will be int32 on 32 bit and int64 on 64 bit

Reply:
done